src/hooks/clipboard.hook.ts

```diff
@@ -0,0 +1,10 @@
+interface ClipboardInterface {
+  copy: (text: string) => Promise<void>;
+}
+
+export function useClipboard(): ClipboardInterface {
+  async function copy(text: string): Promise<void> {
+    console.log('TODO copy', text);
```

Comment:
Show a small notification or temporarily switch the button icon after copy click (maybe for v2, together with spinners 😉).

Reply:
yes totally